### PR TITLE
Carousel Feature

### PR DESCRIPTION
**Pull request by gjuliao**
## Requirement
- Add carousel future to preview cars on Desktop and mobile.
- Fixed footer.

### CODE REVIEW

File: src/components/ProductSlider.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            slidesPerView: 3,
+            spaceBetween: 250,
+          },
+          1280: {
+            slidesPerView: 3,
+            spaceBetween: 300,
+          },
+        }}
+      >
+        <SwiperSlide>
```
Comment (belaymit) [inline src/components/ProductSlider.jsx:39]: I think we can use loop or map method in order to avoid repetition

**Review by belaymit — CHANGES_REQUESTED** [review]
# Status: Changes Required

- [ ] Linters are passing
- [ ] Nice implementation
- [ ] Nice use of methods
However, there are some changes you need to work on: see the comment below

**Comment by gjuliao** [comment]
Hi @belaymit the fetching has been applied within a map function to avoid duplicate code.

Let me know your thoughts on the new design.

**Review by belaymit — APPROVED** [review]
# Status: Approved

- Nice work Giovanni! 
- please merge it

**Comment by gjuliao** [comment]
Awesome @belaymit !

**Pull request merged**